File: Examples/DotNET/CSharp/Mail-Merge/WithoutRegionsWithoutCloudStorage.cs
using System;
using Com.Aspose.Words.Api;
using Com.Aspose.Words.Model;
using Com.Aspose.Storage.Api;

namespace Mail_Merge
{
    class WithoutRegionsWithoutCloudStorage
    {
        public static void Run()
        {
            // ExStart:1
            WordsApi wordsApi = new WordsApi(Common.APP_KEY, Common.APP_SID, Common.BASEPATH);
            StorageApi storageApi = new StorageApi(Common.APP_KEY, Common.APP_SID, Common.BASEPATH);

            String fileName = "SampleMailMergeTemplate.docx";
            String fileDataName = "SampleMailMergeTemplateData.txt";

            String cleanup = null;
            Boolean withRegions = false;

            byte[] file = System.IO.File.ReadAllBytes(Common.GetDataDir() + fileName);
            byte[] data = System.IO.File.ReadAllBytes(Common.GetDataDir() + fileDataName);

            try
            {
                // Invoke Aspose.Words Cloud SDK API to execute mail merge without regions and without using the cloud storage
                ResponseMessage apiResponse = wordsApi.PutExecuteMailMergeOnline(withRegions, cleanup, file, data);


                if (apiResponse != null && apiResponse.Status.Equals("OK"))
                {
                    Console.WriteLine("mail merge online template has been executed successfully");
                    Console.ReadKey();
                }
            }
            catch (Exception ex)
            {
                System.Diagnostics.Debug.WriteLine("error:" + ex.Message + "\n" + ex.StackTrace);
            }
            // ExEnd:1
        }
    }
}

File: Examples/Perl/Mail-Merge/WithoutRegionsWithoutCloudStorage.pl
#ExStart:1
use lib 'lib';
use strict;
use warnings;
use utf8;
use File::Slurp; # From CPAN
use JSON;

use AsposeWordsCloud::WordsApi;
use AsposeWordsCloud::ApiClient;
use AsposeWordsCloud::Configuration;

my $configFile = '../Config/config.json';
my $configPropsText = read_file($configFile);
my $configProps = decode_json($configPropsText);
my $data_path = '../../../Data/';
my $out_path = $configProps->{'out_folder'};

$AsposeWordsCloud::Configuration::app_sid = $configProps->{'app_sid'};
$AsposeWordsCloud::Configuration::api_key = $configProps->{'api_key'};
$AsposeWordsCloud::Configuration::debug = 1;

my $wordsApi = AsposeWordsCloud::WordsApi->new();

# Set input file name
my $name =  'SampleMailMergeTemplate.docx';
my $filedataname = "SampleMailMergeTemplateData.txt";
my $destfilename = "updated-" . $name;

# Invoke Aspose.Words Cloud SDK API to execute mail merge online and populate a word document from XML data      
my $response = $wordsApi->PutExecuteMailMergeOnline(withRegions => 'False', file => $data_path.$name, data=>$data_path.$filedataname);

if($response->{'Status'} eq 'OK'){
	print "\nmail merge online template has been executed successfully.";
	# Download updated document from response
	my $output_file = $out_path. $destfilename;	
	write_file($output_file, { binmode => ":raw" }, $response->{'Content'});
	}
#ExEnd:1